# 客厅
living_room: 
  position: 
    x: 0.888956129551
    y: 0.684198081493
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.186669080588
    w: 0.982422849059

# 卧室
bedroom: 
  position: 
    x: 1.26235949993
    y: -0.381364881992
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.155134544383
    w: 0.987893351096

# 厨房
kitchen: 
  position: 
    x: -1.88215100765
    y: -0.216135337949
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.986761009786
    w: -0.162181101138

# 阳台
balcony: 
  position: 
    x: -1.18901562691
    y: -1.17905819416
    z: 0.0
  orientation: 
    x: 0.0
    y: 0.0
    z: 0.980817486695
    w: -0.194928340151

